alacritty/src/config/mod.rs

```diff
@@ -186,18 +214,17 @@ fn parse_config(
         contents = contents.split_off(3);
     }
 
+    // Convert YAML to TOML as a transitionary fallback mechanism.
+    let extension = path.extension().unwrap_or_default();
+    if (extension == "yaml" || extension == "yml") && !contents.trim().is_empty() {
```

Comment:
From what I remember we don't allow `yaml` extension, just `yml`.

```suggestion
    if (extension == "yml") && !contents.trim().is_empty() {
```

Reply:
Correct, but I intentionally accept both here since `--config-file` is a thing which some people might use in their weird setups. It costs nothing and should be a little safer.

Of course people could also do `--config-file /tmp/whatever`, but if you do that without a file suffix then fuck you, you deserve the error.